server/config/config.go

```diff
@@ -18,6 +18,7 @@ type (
 		GA                 GoogleAnalytics `yaml:"googleAnalytics,omitempty" mapstructure:"googleAnalytics"`
 		Telemetry          Telemetry       `yaml:",omitempty" mapstructure:"telemetry"`
 		Demo               Demo            `yaml:",omitempty" mapstructure:"demo"`
+		Features           Features         `yaml:",omitempty" mapstructure:"features"`
```

Comment:
I think it would be better if this is named `experimentalFeatures`, and it's just a `[]string`, so we don't need to add an extra struct/nesting level